beprof/curve.py

```diff
@@ -5,6 +5,7 @@
 from scipy import signal
 import math
 import copy
+from beprof import functions as f
```

Comment:
Importing style `from x import y as z` was designed to deal with two cases: name clash or very long module names. It is not our case. Read:
- https://code.google.com/p/soc/wiki/PythonStyleGuide#Imports
- http://docs.python-guide.org/en/latest/writing/structure/#modules